TKeazirian.HTTPServer/Handler/SimpleOptionsHandler2.cs

```diff
@@ -0,0 +1,22 @@
+namespace TKeazirian.HTTPServer.Handler;
+
+using Response;
+using Request;
+
+public class SimpleOptionsHandler2 : Handler
+{
+    public override List<string> AllowedHttpMethods()
+    {
+        return new List<string>() { "GET", "PUT", "POST" };
+    }
+
+    public override Response HandleResponse(Request request)
+    {
+        return new ResponseBuilder()
+            .SetStatusCode(HttpStatusCode.Ok)
+            .SetHeaders("Content-Type", "text/plain")
+            .SetHeaders("Content-Length", request.GetRequestBody().Length.ToString())
+            .SetBody(request.GetRequestBody())
+            .Build();
+    }
```

Comment:
are all three methods just going to use the same handle response? this may work for now.  What happens when an endpoint needs to handle multiple methods though?

Reply:
Ah I see...good point.

Reply:
In terms of `PUT` (from RFC: Replace all current representations of the target resource with the request payload.), I'm not sure how to approach it...I think `GET` can just return 200 but I don't think I am grasping how I would differentiate `PUT` from `POST`...any thoughts/hints?

Reply:
It's probably not best to implement this in any way; what does PUT do when the request is properly received but fails? Why no just do that?

Reply:
ref: [HTTP PUT request method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT)

Reply:
How about a [501](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501) ?